src/sentry/models/groupresolution.py

```diff
@@ -21,9 +21,9 @@ class Status:
         resolved = 1
 
     group = FlexibleForeignKey('sentry.Group', unique=True)
-    # the release in which its suggested this was resolved
+    # the release which is suggested to resolve the issue
     # which allows us to indicate if it still happens in newer versions
-    release = FlexibleForeignKey('sentry.Release')
+    release = FlexibleForeignKey('sentry.Release', null=True)
```

Comment:
Should there be a generated migration with this?

Reply:
oops this wasn't supposed to make it in -- will back out once we decide on api input/output